.github/workflows/clickhouse-importer-docker-image.yml

```diff
@@ -0,0 +1,46 @@
+name: Update Clickhouse Importer Docker Image
+
+on:
+  push:
+    paths:
+      - 'addon/clickhouse/importer.Dockerfile'
+      - '.github/workflows/clickhouse-importer-docker-image.yml'
+
+jobs:
+  update-clickhouse-importer-image:
+    runs-on: ubuntu-latest
+    if: github.repository == 'cbioportal/cbioportal-docker-compose'
+    env:
+      DOCKERHUB_REPO: 'cbioportal/clickhouse-importer'
+      PLATFORMS: 'linux/amd64,linux/arm64'
+      GIT_OWNER: 'sheridancbio'
```

Comment:
If you can wait a day or so to merge I can try to push the Tools PR to completion so that this can come from the cbioportal-core repo (once the other PR is merged). 
Or, if you want to move forward today, I could squash work in that repo as is (with the one bug fix) and do the documentation additions later as a separate PR.